src/components/LayoutHelpers.tsx

```diff
@@ -5,7 +5,7 @@ import { useUserState } from '../contexts/UserState'
 import strings from '../utils/constants'
 import usePageTitle from '../utils/hooks/usePageTitle'
 
-const Home: React.FC = () => {
+const LayoutHelpers: React.FC = () => {
```

Comment:
Can you update the page name for this one (line 18): `usePageTitle("Layout Helpers")`

Reply:
b6f2e2b5ff34f616824c9adfc6cf9113eebc30e1